### PR TITLE
[FrameworkBundle][TranslationUpdateCommand] Do not output positive feedback on stderr

### PR DESCRIPTION
**Pull request by fancyweb**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | https://github.com/symfony/symfony/pull/34865#discussion_r362432040
| License       | MIT
| Doc PR        | -

Outputing everything on stderr seems wrong. I'm going to check upper branches.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Maybe not relevant for this specific command, but outputting informational messages on stderr is often quite legit: when the command produces technically useful output (compared to info messages which are useful only for human readers), then these should be split to stdout/stderr.

File: src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } elseif ($tag = $input->getOption('tag')) {
             $options = ['tag' => $tag, 'show_private' => $input->getOption('show-private')];
         } elseif ($name = $input->getArgument('name')) {
-            $name = $this->findProperServiceName($input, $errorIo, $object, $name);
+            $name = $this->findProperServiceName($input, $io, $object, $name);
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php]: This one looks wrong to me, see my comment on the PR.

**Review by nicolas-grekas — COMMENTED** [review]
Why this PR?

**Comment by fancyweb** [comment]
> Why this PR?

Because all other commands output this kind of things on stdout which seems better to me. Let's be consistent.

File: src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php (hunk 1 above)
Comment (fancyweb) [reply src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php]: This method ask the user to select a service. In 2 other commands we do it on stdout. So I don't see why it needs to be different here.

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php]: This command handles multiple output formats including machine-readable ones. It uses stdout for contents that are meant to be redirected. Interactions are informational contents, nothing processable, hence they should go to stderr (e.g. to not pollute a xml result).

Comment (fancyweb) [reply src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php]: Thanks for the explanation, I reverted. As an example, if you do `php bin/console debug:container webpack_encore --format=xml > foo.xml`, the choice selection is not shown if we output it on stdout.

**Review by nicolas-grekas — APPROVED** [review]

**Review by yceruto — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @fancyweb.

**Pull request merged**